app/services/domain/claim-decision.js

```diff
@@ -69,6 +69,23 @@ class ClaimDecision {
           .isLessThanMaximumDifferentApprovedAmount()
       }
     })
+    var check = false
+    this.claimExpenseResponses.forEach(function (expense) {
+      if (expense.status.includes(claimDecisionEnum.APPROVED)) {
+        check = true
+      } else if (expense.status.includes(claimDecisionEnum.APPROVED_DIFF_AMOUNT)) {
+        check = true
+      } else if (expense.status.includes(claimDecisionEnum.MANUALLY_PROCESSED)) {
+        check = true
+      } else if (expense.status.includes(claimDecisionEnum.REQUEST_INFORMATION)) {
+        check = true
+      }
+    })
+
+    if (check === false) {
+      FieldValidator(this.approvedCost, 'approve-cost', errors)
+        .isRequired('', check)
+    }
```

Comment:
Error message displayed on the page if all expense responses are rejected is `New approved cost is required`

It should be something like:

`At least one expense must not be rejected for the claim to be approved`.

We could just add the error directly in here instead of using the field validator.

Something like

`errors.add('claim-expenses', ERROR_MESSAGES.getNonRejectedClaimExpenseResponse)`